src/lib/models/cylinder/index.tsx

```diff
@@ -38,6 +53,40 @@ export class Cylinder extends Model<
 	declare createdAt: CreationOptional<Date>
 	// updatedAt can be undefined during creation
 	declare updatedAt: CreationOptional<Date>
+
+	declare getOwner: HasOneGetAssociationMixin<Client> // Note the null assertions!
```

Comment:
I dont really like how v6 does this but v7 isnt stable yet, so I suppose we wait